consensus/serde_utils/src/quoted_u64_fixed_vec.rs

```diff
@@ -0,0 +1,129 @@
+//! Formats `FixedVector<u64,N>` using quotes.
+//!
+//! E.g., `FixedVector::from(vec![0, 1, 2])` serializes as `["0", "1", "2"]`.
+//!
+//! Quotes can be optional during decoding. If `N` does not equal the length of the `Vec`, the `Vec` is truncated.
+
+use serde::ser::SerializeSeq;
+use serde::{Deserializer, Serializer};
+use serde_derive::{Deserialize, Serialize};
+use ssz_types::typenum::Unsigned;
+use ssz_types::FixedVector;
+use std::marker::PhantomData;
+
+#[derive(Serialize, Deserialize)]
+#[serde(transparent)]
+pub struct QuotedIntWrapper {
```

Comment:
It's pretty minor, but perhaps we could import this from `crate::quoted_u64_vec`?